cylc/flow/pathutil.py

```diff
@@ -88,26 +88,26 @@ def get_workflow_run_job_dir(workflow, *args):
     return get_workflow_run_dir(workflow, 'log', 'job', *args)
 
 
-def get_workflow_run_log_dir(workflow, *args):
+def get_workflow_run_scheduler_log_dir(workflow, *args):
     """Return workflow run log directory, join any extra args."""
-    return get_workflow_run_dir(workflow, 'log', 'workflow', *args)
+    return get_workflow_run_dir(workflow, 'log', 'scheduler', *args)
 
 
-def get_workflow_run_log_name(workflow):
+def get_workflow_run_scheduler_log_name(workflow):
     """Return workflow run log file path."""
-    return get_workflow_run_dir(workflow, 'log', 'workflow', 'log')
+    return get_workflow_run_dir(workflow, 'log', 'scheduler', 'log')
```

Comment:
Yes, I think there appears to be a convention in the codebase to end with name when it returns the full path, e.g. `get_workflow_run_pub_db_name`, `get_workflow_test_log_name` return the full path rather than just the name. I'll change these to end in path rather than name, for clarity.